frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -103,7 +103,9 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
               <SplitItem>
                 <Bullseye>
                   <TextContent>
-                    <TextListItem component={TextListItemVariants.dt}>Container</TextListItem>
+                    <TextListItem component={TextListItemVariants.dt}>
+                      {podContainers.length === 0 ? 'Step' : 'Steps'}
```

Comment:
@yih-wang I saw in the [mocks](https://www.figma.com/proto/rJjOtl3nxpW6BAcl0MZr4y/Pipeline-log-viewer?page-id=62%3A1895&type=design&node-id=260-2386&viewport=664%2C-1095%2C0.23&t=80ExnogduAcwD3kx-1&scaling=min-zoom&starting-point-node-id=260%3A2386) that we see "Steps" when there are multiple steps and "Step" when there is one. Is this intentional? Just confirming coz I didn't see any comments regarding this in the mocks.

Reply:
Interesting... I always assumed this was "Step <blank>" "Container <blank>"... is this intended as a label for the dropdown? "Steps" "Containers" as in "there are multiple involve here"?

Reply:
@andrewballantyne No, it's not intentional. The only reason for the difference is my bad English... I believe the label should stay consistent, whether there's one or more options in the dropdown. I have no idea about whether to use the singular or plural form in this context. @kaedward, could you help on this matter?

Reply:
But I guess the singular form (Step) is the answer according to what we have for the other dropdown labels around the console.

Reply:
Ah gotcha... thanks for the clarification. @manaswinidas implement just singular.